### DISCUSSION TITLE
DATABASE_URL Connector Not working with Instance Name

### DISCUSSION THREAD

**Discussion by Acurthwilliams** [prisma/prisma#15754] [Q&A] [answered]
Hi the other day our Prisma connector which had the Instance name in the connection string stopped working. We resolved this by removing the instance name from the connection string. I'm wondering if there was any changes made that may've caused this functionality to no longer work.

Previous connection string - not working
DATABASE_URL=sqlserver://serverIP\instance;database=db;user=sa;password=123abc;trustServerCertificate=true

New connection string - working
now DATABASE_URL=sqlserver://serverIP;database=dbuser=sa;password=123abc;trustServerCertificate=true

OS: Linux
Database: MSSQL

**Comment by nurul3101** [comment]
Hi @Acurthwilliams 👋 
Had you updated the Prisma version? The changes even if there are any should only take place when you change the Prisma version.
I am thinking that maybe certain changes deployed by the database provider could have triggered this?

**Comment by Acurthwilliams** [comment, marked as answer]
Hi nurul, Our Dev said our version is Prisma was locked and had not changed. Also there were no changes made to the DB host/provider.

I realise I didn't mention in my original post that our instance is containerised within Azure. Is it possible or likely even then that there was a platform change in Azure that impacted this functionality?

**Reply by nurul3101** [reply]
It definitely could be a possibility, I haven't seen someone reporting this kind of behaviour before. Maybe open a [Bug Report](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml) so that our engineering team can also have a look?

**Reply by vladi-stevanovic** [reply]
👋 Hi @Acurthwilliams did you perhaps have a chance to open a bug already?
Please let us know so that we can keep track of this issue and our Eng team can continue investigating it!

**Reply by Acurthwilliams** [reply]
Hi we had a similar issue on another Azure platform not using Prisma so I expect the issue is related to the underlying platform rather than the Prisma software so I've elected not to raise a bug request.

**Reply by vladi-stevanovic** [reply]
Thank you for letting us know! If you indeed determine that this is Prisma related, don't hesitate to follow up here ! 💯

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
